server/package.json

```diff
@@ -5,6 +5,9 @@
   "author": "",
   "private": true,
   "license": "UNLICENSED",
+  "engines": {
+    "node": ">=16"
```

Comment:
Just double checking - the `>=` here limits accepted versions to 16.x, correct? As in, 18 wouldn't be excepted based on this, right?

Reply:
ooof, not sure how I made that typo in one place but not the other. In that case, can you change this to just `"^16"`? That _should_ translate to "anything 16.0.0 or above but below 17.0.0". Great to hear 18 works but having a value there that doesn't at least pin the major version seems to be asking for problems down the road.